actions/create-task/action.php

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/php
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+use onOffice\SDK\onOfficeSDK;
+
+$optionsConfig = [
+    'api-version::',        // string [stable|latest] (default: latest)
+    'subject:',             // string (required)
+    'text:',                // string (required)
+    'type:',                // int (required)
+    'responsible:',         // string (required)
+    'responsibleIsGroup::', // int [0|1] (default: 1)
+    'editor::',             // string
+    'project::',            // int
```

Comment:
This is fine, but `project` could also be a comma-separated list of project IDs. I don't really see a reason to artificially only allow one linked project.